### PR TITLE
Add create execution function

### PR DESCRIPTION
**Pull request by vermakhushboo**
## What does this PR do?

Update to stable version for 1.5.x and add create execution function test

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -652,6 +664,19 @@ <h2 class="mb-5">Locale</h2>
...
+            promise.then(
+                function (response) {
+                    alert("Function executed successfully!");
+                },
+                function (error) {
+                    alert("Failed to execute function.");
+                }
+            );
+        }
+
```
Comment (christyjacob4) [inline public/index.html]: @vermakhushboo there should also be code to create the deployment and activate it

Comment (vermakhushboo) [reply public/index.html]: I don’t think web SDK allows user to create function or deployment. It only has create execution and list executions methods. So we need to create that function and deployment manually to test this flow.

Comment (vermakhushboo) [reply public/index.html]: Added comment explaining how to create function

**Comment by vermakhushboo** [comment]
Closing as created another PR for 1.6 changes

**Pull request closed without merging**